pipeline.py

```diff
@@ -2,6 +2,9 @@
 Pipeline for text processing implementation
 """
 
+from constants import ASSETS_PATH
```

Comment:
as soon as target score file does not contain a positive mark, tests are not run for your pipeline implementation. put there a mark otherwise there might be some unexpected surprises when you finally do that

Reply:
Fixed